sycl/source/detail/scheduler/commands.hpp

```diff
@@ -104,13 +106,15 @@ class Command {
 
   void addUser(Command *NewUser) { MUsers.insert(NewUser); }
 
-  // Return type of the command, e.g. Allocate, MemoryCopy.
+  /// \return type of the command, e.g. Allocate, MemoryCopy.
   CommandType getType() const { return MType; }
 
-  // The method checks if the command is enqueued, waits for it to be
-  // unblocked if "Blocking" argument is true, then calls enqueueImp. Returns
-  // true if the command is enqueued. Sets EnqueueResult to the specific
-  // status otherwise.
+  /// Checks if the command is enqueued, and calls enqueueImp.
+  ///
+  /// \param EnqueueResult is set to the specific status if enqueue failed.
+  /// \param Blocking if this argument is true, function will wait for command
```

Comment:
There are multiple inconsistencies between articles' usage (`command` here, `the command` at line 112). Nitpicking on each of these wouldn't be wise, but a quick revision would be appreciated. In this context, the article is often required.